modules/create-vm/go.mod

```diff
@@ -7,18 +7,20 @@ require (
 	github.com/kubevirt/kubevirt-tekton-tasks/modules/shared v0.0.0
 	github.com/kubevirt/kubevirt-tekton-tasks/modules/sharedtest v0.0.0
 	github.com/mattbaird/jsonpatch v0.0.0-20171005235357-81af80346b1a
-	github.com/onsi/ginkgo/v2 v2.1.6
-	github.com/onsi/gomega v1.20.1
+	github.com/onsi/ginkgo/v2 v2.2.0
+	github.com/onsi/gomega v1.20.2
 	github.com/openshift/api v0.0.0
 	github.com/openshift/client-go v0.0.0
+	github.com/spf13/cobra v1.5.0
 	go.uber.org/zap v1.19.1
 	k8s.io/api v0.25.2
 	k8s.io/apimachinery v0.25.2
 	k8s.io/client-go v12.0.0+incompatible
-	kubevirt.io/api v0.59.0
-	kubevirt.io/client-go v0.59.0
+	kubevirt.io/api v0.60.0-alpha.0
```

Comment:
Can we stay on stable?

Reply:
Ondrej select this tag, because there was something missing in 0.59. @opokornyy can you please confirm?

Reply:
Yes, the stable tag does not contain all of the virtctl functionality that is needed.

Reply:
For the latest fixes you potentially want to pin the version to a specific commit. E.g. to include https://github.com/kubevirt/kubevirt/pull/9651

Reply:
You might want to update this to `v1.0.0-alpha.0` to have https://github.com/kubevirt/kubevirt/pull/9651 included.